ndsi/__init__.py

```diff
@@ -24,6 +21,13 @@ def __init__(self, message):
         self.message = message
 
 
+from ndsi.formatter import DataFormat
+
+
+__version__ = "0.6"
+__protocol_version__ = str(DataFormat.latest().version_major)
```

Comment:
I have the feeling, we should remove this, even though it would be nice to have it for backwards compatibility.

Reply:
I think it's not very meaningful to have this, since you can explicitly specify which data format version(s) to use. And if you do specify a version different from the default `DataFormat.latest()`, then `__protocol_version__` no longer reflects the truth. Right now it's useful for testing this PR with existing projects.

I think the best approach would be:
1. keep `__protocol_version__` it in this PR
2. merge the PR and release a new version of `ndsi`
3. update all the projects that use `ndsi` to this new version of the library and remove any uses of `__protocol_version__`
4. remove `__protocol_version__` from `ndsi` in a separate PR

Let me know how you want to proceed.

Reply:
I agree on the procedure.